node/node.go

```diff
@@ -1224,6 +1223,7 @@ func (node *AlgorandFullNode) SetCatchpointCatchupMode(catchpointCatchupMode boo
 		node.blockService.Start()
 		node.ledgerService.Start()
 		node.txHandler.Start()
+		node.stateProofWorker.Start()
```

Comment:
I think we should rearrange the order of the services in`Start` and `Stop` in `AlgorandFullNode.Start`/`Stop`. i.e If the state proof service is the last service to be initiated (it is also dependent on the ledger, network etc) it should be the first one to be stopped